app/helpers/validate/throwMissingParameter.spec.js

```diff
@@ -0,0 +1,48 @@
+const assert = require("assert")
+const throwMissingParameter = require("./throwMissingParameter");
+
+describe("helpers/validate/throwMissingParameter", () => {
+    describe("fields should match object keys", () => {
+        it("matching keys", () => {
+            const object = {
+                keyone: "value",
+                keytwo: "another value"
```

Comment:
Codacy has a fix for the issue: [Strings must use singlequote.](https://app.codacy.com/gh/ufabc-next/ufabc-next-server/pullRequest?prid=8555381)
```suggestion
                keytwo: 'another value'
```